front/src/components/App/MaintenanceArea/AuctionSitesManagement.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from "react";
+import AuctionSitesList from "./AuctionSitesList";
+import AuctionSitesAddition from "./AuctionSitesAddition";
+
+export interface AuctionSite {
+  name: string;
+}
+
+export interface AuctionSitesInterface {
+  setAuctionSites: React.Dispatch<React.SetStateAction<AuctionSite[]>>;
+  auctionSites: AuctionSite[];
+}
+
+export interface FormValues {
+  setCurrentName: React.Dispatch<React.SetStateAction<string>>,
+  currentName: string
+}
+
+const AuctionSitesManagement = () => {
+
+  const [inputName, setInputName] = useState("")
+  const [auctions, setAuctions] = useState<AuctionSite[]>([])
+
+
+  return (
+      <div>
+
+        <AuctionSitesAddition auctionInterface={{setAuctionSites: setAuctions, auctionSites: auctions}}
```

Comment:
dlaczego w taki sposób tutaj są przekazywane propsy? Jeśli nie ma konkretnego powodu to zwykle lepiej przesyłać je pojedynczo. Szczególnie że tutaj ich dużo nie ma, a dodatkowe nazwy wproawdzają wiecej chaosu